test/qtismtest/runtime/common/OrderedContainerTest.php

```diff
@@ -12,32 +12,37 @@
 use qtism\runtime\common\MultipleContainer;
 use qtism\common\enums\BaseType;
 
-class OrderedContainerTest extends QtiSmTestCase {
+class OrderedContainerTest extends QtiSmTestCase
+{
 	
 	/**
 	 * @dataProvider equalsValidProvider
 	 */
-	public function testEqualsValid($containerA, $containerB) {
+	public function testEqualsValid($containerA, $containerB)
+    {
```

Comment:
Wrong indent in L22 L31, L37 and L45. Maybe because of `space` vs `tab`

Reply:
Thanks @peetya ! solved